docs/greptimecloud/integrations/fluent-bit.md

```diff
@@ -0,0 +1,66 @@
+---
+keywords: [Fluent Bit, GreptimeCloud, metrics ingestion, logs ingestion, data pipeline]
+description: Guide for using Fluent Bit with GreptimeCloud, including configuration for metrics and logs ingestion, and running Fluent Bit with a sample configuration.
+---
+
+
+# Fluent Bit
+
+Fluent Bit is a lightweight and fast log processor and forwarder that can collect, parse, filter, and forward logs and metrics. Fluent Bit is part of the Fluentd project ecosystem and is written in C language. It is designed to be memory-efficient and performant, making it suitable for use in resource-constrained environments.
+
+## Prometheus Remote Write
+
+Fluent Bit can be configured to send metrics to GreptimeCloud using the Prometheus Remote Write protocol. This allows you to collect metrics from various sources and send them to GreptimeCloud for storage, analysis, and visualization.
+
+```
+[OUTPUT]
+    Name                 prometheus_remote_write
+    Match                internal_metrics
+    Host                 <host>
+    Port                 443
+    Uri                  /v1/prometheus/write?db=<dbname>
+    Tls                  Off
+    http_user            <username>
+    http_passwd          <password>
+```
+
+In this example, the `prometheus_remote_write` output plugin is used to send metrics to GreptimeCloud. For more information, and extra options, refer to the [Prometheus Remote Write](https://docs.greptime.com/user-guide/integrations/prometheus) guide.
+
+## OpenTelemetry
+
+Fluent Bit can be configured to send logs and metrics to GreptimeCloud using the OpenTelemetry protocol. This allows you to collect logs and metrics from various sources and send them to GreptimeCloud for storage, analysis, and visualization.
+
+```
+# Only for metrics
+[OUTPUT]
+    Name                 opentelemetry
+    Alias                opentelemetry_metrics
+    Match                *_metrics
+    Host                 <host>
+    Port                 443
+    Metrics_uri          /v1/otlp/v1/metrics
+    Logs_uri             /v1/otlp/v1/logs
+    Traces_uri           /v1/otlp/v1/traces
```

Comment:
logs_uri and traces_uri looks confusing when it's only for metrics

Reply:
This is only a soft limitation, it is possible for the user to write two other types of data through this output, so both trace and log configurations are kept here.

Reply:
For traces we will require additional headers configuration. So it's better to remove this to prevent user from copying it to traces.